pkg/remote/aws/init_test.go

```diff
@@ -1,14 +1,16 @@
 package aws
 
 import (
+	"os"
+
 	"github.com/cloudskiff/driftctl/pkg/output"
 	"github.com/cloudskiff/driftctl/pkg/terraform"
 )
 
 func InitTestAwsProvider(providerLibrary *terraform.ProviderLibrary) (*AWSTerraformProvider, error) {
 	progress := &output.MockProgress{}
 	progress.On("Inc").Maybe().Return()
-	provider, err := NewAWSTerraformProvider("", progress, "")
+	provider, err := NewAWSTerraformProvider("3.19.0", progress, os.TempDir())
```

Comment:
May be we should put this in a constant ?

Reply:
I don't think that is has the same sense that in the part of the code where we assign it as a default value
This value should be enforced by the test and does not have to follow the "latest default" from the code or it will break some test. This need to be reproductible